tests/virt_autotest/libvirt_virtual_network_init.pm

```diff
@@ -45,31 +45,30 @@ sub run_test {
     virt_autotest::virtual_network_utils::enable_libvirt_log();
 
     #VM HOST SSH SETUP
-    virt_autotest::virtual_network_utils::ssh_setup();
+    virt_autotest::virtual_network_utils::ssh_setup() if (!(get_var('VIRT_AUTOTEST')));
 
     #Backup file /etc/hosts before virtual network testing
     virt_autotest::virtual_network_utils::hosts_backup();
 
     #Install required packages
     zypper_call '-t in iproute2 iptables iputils bind-utils sshpass nmap';
 
-    #Check with Guest status before libvirt virtual network tests
-    virt_autotest::virtual_network_utils::check_guest_status();
-
```

Comment:
I don't understand this either.

Reply:
In the past, after finished guest installation, all deployed guest system will be set as the offline status, check_guest_status func just double check guest system as offline, then to use dumpxml to create offline guest configure file for restore_guests func. 
So far, after moved virtual network under setup_dns_service, all guest system set the **online** status, also our virtual network test required all deployed guest system as online status.
So, following the changes from setup_dns_service, we just change to create the **online** guest configure file for restore_guests func. means that we do not need to shutdown guest, create offline guest configure file and then reboot up guest again. 
So, remove the check_guest_status func here.